### PR TITLE
added boxcolor to geom_label

### PR DESCRIPTION
**Pull request by TyberiusPrime**
Improvement closing #225

### CODE REVIEW

File: plotnine/geoms/geom_label.py [hunk 1]
```diff
@@ -15,6 +15,7 @@
     #
     # Documentation at matplotlib.patches.BoxStyle
     'boxstyle': 'round',
+    'boxcolor': 'black',
```
Comment (has2k1) [inline plotnine/geoms/geom_label.py]: Set the default to `None` so that `boxcolor` only has an effect if the user has set it.

**Comment by has2k1** [comment]
Avoid my `dev` branch. I rebase often.

File: plotnine/geoms/geom_label.py (hunk 1 above)
Comment (TyberiusPrime) [reply plotnine/geoms/geom_label.py]: I've restored the old behaviour: unset boxcolor means 'use color aes' and documented accordingly.

**Comment by has2k1** [comment]
Thanks. I did some amending, they are now in master.

**Pull request closed without merging**